cluster-logging/overlays/moc/infra/externalsecrets/clo-kafka.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: clo-kafka
+spec:
+  secretStoreRef:
+    name: opf-vault-store
+    kind: SecretStore
+  target:
+    name: clo-kafka
+  dataFrom:
+    - extract:
+        key: moc/infra/openshift-logging/clo-kafka
```

Comment:
This won't work, since the crt content has been stored as binaries, you need to do something like this: 

https://github.com/operate-first/apps/blob/master/observatorium/overlays/moc/smaug/vector/clo-kafka.yaml#L14-L18
which gets data from: https://vault-ui-vault.apps.smaug.na.operate-first.cloud/ui/vault/secrets/k8s_secrets/show/moc/smaug/opf-observatorium/clo-kafka

Reply:
same goes for smaug/externalsecrets